d2cli/static/watch.js

```diff
@@ -73,3 +74,18 @@ function init(reconnectDelay) {
     }, reconnectDelay);
   };
 }
+
+const changeFavicon = function (iconURL) {
+  const head = document.getElementsByTagName("head")[0];
+  const newLink = document.createElement("link");
+  newLink.rel = "icon";
+  newLink.href = iconURL;
+  removeExistingFavicons();
+  head.appendChild(newLink);
```

Comment:
do you have o remove then append? can you not just get the existing element and set the `href` attribute?

Reply:
@alixander sorry for the delay.
I totally agree with you.
I updated my code.
Please recheck it when you got time.